components/base/sensorcontrolled/spin.go

```diff
@@ -49,6 +49,7 @@ func (sb *sensorBase) Spin(ctx context.Context, angleDeg, degsPerSec float64, ex
 			return err
```

Comment:
can't figure out how to leave a comment on it, but above this in line 42, do you think we should pause the loop there? or continue to stop it. same goes for the other files. basically in any case where a sensor is unsupported, should we pause the loop or stop it

Reply:
pause the loop imo. Unsupported apis would effectively be treated like SetPower calls